src/utils/validations.js

```diff
@@ -0,0 +1,24 @@
+const isValidHour = hour => typeof hour === 'number' && hour >= 0 && hour < 24
+
+const isValidMinute = min => typeof min === 'number' && min >= 0 && min < 60
+
```

Comment:
1. A Validation function doesn't return any data. Change the name to something that reflect that you are also transforming, like validateAndTransformHours.
2. If you are not going to operate with arrays, don't use them. Return an object.